testing/single/src/formatOnType.py

```diff
@@ -20,4 +20,22 @@
 if ch == "a":
     pass
     # type `:` after `else` and see `else` is moved to right position.
-    else
\ No newline at end of file
+    else
+    
+
+# Set "autoFormatStrings" to true and add a { into the string below. Make sure
+# it auto adds the `f` on the front. Repeat for the other strings
+x = ""
+y = ''
+z = """""""
+w = ''''''
+
+# Make sure it doesn't add the `f` if it's already there
+a = f""
+b = f''
+c = f"""""""
+d = f''''''
+
+# Make sure it doesn't add the `f` if it the start is a r or b string.
+e = r""
```

Comment:
`rf` is supported by Python btw.

Reply:
Oh well i guess we should support it too. It's not currently supported by our algorithm though.

Reply:
Added an issue https://github.com/microsoft/pylance-release/issues/5703